db/migrate/20230923154144_create_tasks.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTasks < ActiveRecord::Migration[6.1]
+  def change
+    create_table :tasks do |t|
+      t.string :title, null: false
+      t.text :description
+      t.boolean :state, default: false
+      t.datetime :deadline
```

Comment:
Usually the date and time fields should be named with the `at` postfix 

```suggestion
      t.datetime :deadline_at
```